### ISSUE TITLE
arm64 docker image does not work for 0.13.2

### ISSUE THREAD

**Issue by githubcdr** [metallb/metallb#1488]
```
helm -n metallb upgrade -i metallb metallb/metallb --set controller.image.tag=0.13.2 
```

Does not work on arm64 hosts, the wrong manifest gets pulled resulting in "exec /speaker: exec format error"

**Comment by fedepaol** [comment]
Thanks for the report, and sorry for the inconvenience. We are aware of the issue, and we are planning to craft a new release by the end of the day.
Closing as duplicate of https://github.com/metallb/metallb/issues/1476

**Comment by fedepaol** [comment]
@githubcdr just released 0.13.3, which should fix arm support
